build-library/action.yml

```diff
@@ -0,0 +1,54 @@
+name: "Build a Python library"
+description: "Build a Python library by generating wheels and sdist artifacts"
+
+inputs:
+  library-name:
+    description: 'Name of the Python library'
+    required: true
+    type: string
+  python-version:
+    description: 'Desired Python version for pre-commit'
+    default: '3.10'
+    required: false
+    type: string
+
+runs:
+  using: "composite"
+  steps:
+  
+    - name: "Install Git and clone project"
+      uses: actions/checkout@v3
+  
+    - name: "Setup Python"
+      uses: actions/setup-python@v4.2.0
+      with:
+        python-version: ${{ inputs.python-version }}
+
+    - name: "Check if requirements_build.txt file exists"
+      shell: bash
+      run: |
+        echo "EXISTS_BUILD_REQUIREMENTS=$(if [ -f requirements/requirements_build.txt ]; then echo 'true'; else echo 'false'; fi)" >> $GITHUB_ENV
```

Comment:
Unfortunately, Sometimes the  `requirements_*.txt` files are not located into the `requirements` folder but directly at the root of the repo.
We could also check for that.

Reply:
Well that is definitely something that has to be solved on a per  repo basis... It is one of the basic recommendations. Whoever still has it that way, we can open a PR to solve it.

Reply:
I guess you raised this concern since PyAEDT collects its requirements at base root directory level, @MaxJPRey. It should be feasible for us, as @RobPasMue, to open a pull-request and move those to a requirements directory.